scripts/scil_surface_create.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to create a surface with marching cube from a mask or a label image.
+The surface will be readable with software like MI-Brain.
+
+Example : use wmparc.a2009s.nii.gz with some aseg.stats indices
+
+scil_surface_create.py out_surface.vtk \\
+    --in_labels s1a1/mask/S1-A1_wmparc.a2009s.nii.gz\\
+    --indices 16:32 --vox2vtk --opening 2 --smooth 2 -v
+"""
+
+import argparse
+import logging
+
+import nibabel as nib
+import trimeshpy
+import trimeshpy.vtk_util as vtk_u
+from scipy.ndimage import (binary_closing,
+                           binary_dilation,
+                           binary_erosion,
+                           binary_opening,
+                           binary_fill_holes)
+import mcubes
+
+from scilpy.image.labels import (get_data_as_labels,
+                                 merge_labels_into_mask)
+from scilpy.io.image import get_data_as_mask
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_verbose_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             ranged_type)
+
+EPILOG = """
+References:
+[1] St-Onge, E., Daducci, A., Girard, G. and Descoteaux, M. 2018.
+    Surface-enhanced tractography (SET). NeuroImage.
+"""
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__, epilog=EPILOG,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    g = p.add_argument_group("Input (Labels or Mask)")
+    mxg = g.add_mutually_exclusive_group(required=True)
+    mxg.add_argument('--in_labels',
+                     help='Path of the atlas (nii or nii.gz). '
+                     'It will use the indices provided with --indices. '
+                     'If no indices are provided, it will use all indices.')
```

Comment:
"it will use all indices." is not completely true, since it takes all positives  >= 1 values.
It could be rewritten to "it use all positive indices".
(Yes some label volumes have negative values, some freesurfer labels too; but no scilpy does not seems to support it since label_volume are loaded as uint16 in get_data_as_labels() )